frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/CreateEstimate/CreateEstimateForm.js

```diff
@@ -108,15 +100,25 @@ const CreateEstimateForm = ({ onFormSubmit }) => {
 
 
     const { subTypes: SubTypeOfWork } = useWatch({ control: control, name: "typeOfWork", defaultValue: [] });
+    SubTypeOfWork?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.code}`)}))
 
     if (data?.works) {
         var { EntrustmentMode, BeneficiaryType, NatureOfWork, TypeOfWork } = data?.works
     }
+    EntrustmentMode?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.code}`)}))
+    BeneficiaryType?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.code}`)}))
+    NatureOfWork?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.code}`)}))
+    TypeOfWork?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.code}`)}))
 
     const { subSchemes: subScheme } = useWatch({ control: control, name: "scheme", defaultValue: [] });
     if (financeData?.finance) {
         var { Scheme, BudgetHead, Functions, Fund } = financeData?.finance
     }
+    Scheme?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.schemeCode}`)}))
+    BudgetHead?.map((item)=> Object.assign(item, {i18nKey:t(`ES_COMMON_${item?.code}`)}))
```

Comment:
Budget head is just a code so please add prefix like ES_COMMON_BUDGETHEAD_code. Otherwise it'll be ES_COMMON_01,ES_COMMON_02 and that will cause issues